eslint.config.js

```diff
@@ -19,11 +19,26 @@ const compat = new FlatCompat();
 module.exports = [
   {
     ignores: [
-      'examples/',
-      'codelabs/',
-      'gh-pages/',
```

Comment:
I think we want to exclude `gh-pages` still as that is largely autogenerated. It may not error now but it would be annoying if something generated warnings later. I'd also probably ignore `codelabs` since they're still ignored by prettier.